TACC/Core-CMS-Custom
# feat: ecep_cms

## Overview

Migrate project from Core-CMS-Resources.

> **Important**
> Delete the "CSS: Site" snippet ([pprd](https://pprd.ecep.tacc.utexas.edu/admin/djangocms_snippet/snippet/10/change/), [prod](https://ecepalliance.org/admin/djangocms_snippet/snippet/10/change/)) to remove the "Template ecep-cms/templates/snippets/site-css.html does not exist." error from the footer.
> <details><summary>Screenshot</summary>
> <img width="992" alt="snippet error message" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/d2d94052-e56c-47c4-8bbb-e46ece238ad8">
> </details>

## Related

- (after this current pr) this can benefit from #209

## Testing

1. Verify [deploy](https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/3170/) succeed. (https://pprd.ecep.tacc.utexas.edu/)
2. Verify custom CSS loads. (It gives purple header and links.)

## UI

<details><summary>Many Screenshots</summary>

- <img width="1200" alt="version" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/8973ffd3-785b-4423-84de-0f7d41e55097">
- <img width="1200" alt="home top" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/d36aaf15-0e6b-48c1-97bf-3dbccc516328">
- <img width="992" alt="home bottom" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/c8074f96-1493-41ae-914e-e6bd12355f62">
	👆 To remove the error at bottom of footer, delete that snippet. 👆
- <img width="1200" alt="alliance-members" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/77b7dc33-23bb-4468-8f6f-0571ec7378d5">
- <img width="1200" alt="news list" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/15ab5f95-61bf-4e51-80d9-f1e70e75e804">
- <img width="1200" alt="news page" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/a60e82da-c6d3-472a-b2db-d5e2af9908eb">
- <img width="1200" alt="resources" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/ced2a0d4-77e4-4948-a7f0-fc014c98e719">
- <img width="1200" alt="publications" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/12f1e5bb-fd76-47be-99d2-2e98d5743d66">

</details>
